spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+RSpec.describe UsersController, type: :controller do
+
+  describe "#homepage" do
+    context "with user signed in" do
+      let!(:user) { FactoryGirl.create( :user ) }
```

Comment:
You can move this to a `before(:each)` block on the describe block, as you're using it in both context